modules/widgets/google-plus/js/admin.js

```diff
@@ -0,0 +1,17 @@
+(function($) {
+	$(function(){
+		$( document ).on( 'change', '.googleplus-badge-choose-type', toggle_items )
+			.on( 'widget-updated', toggle_items );
+
+		toggle_items();
+	});
+
+	function toggle_items() {
+		$( '.widget-inside .googleplus-badge-choose-type' ).each( function(){
+			var $widget_form = $( this ).parents( 'form' );
+
+			$widget_form.find( "[class^='googleplus-badge-only-']" ).parent().hide();
```

Comment:
our js linter is very picky about quotes.  Can you please inverse the double/single quotes like 
`'[class^="googleplus-badge-only-"]'`?